README.md

```diff
@@ -80,7 +80,7 @@ will be true in GHC 8.10.1.
 ## Examples
 
 Make sure you read the [Necessary Language
-Extensions](https://github.com/isovector/polysemy#necessary-language-extensions)
+Extensions](https://github.com/isovector/polysemy#necessary-language-extensions) and add any extras in example source blocks (such as LambdaCase and BlockArguments below)
```

Comment:
I really don't understand this sentence, sorry! Can you explain it to me? Mostly, what are the "extras" referring to?

Reply:
extras == extra language pragmas

Reply:
Yeah, but what are they extra in reference to?
Do you mean "add any that are listed in the necessary extensions, but not enabled in the example source block"?